assets/questions/i18n.js

```diff
@@ -70,9 +75,9 @@ def createListOfUniqueStrings(atom, size):
       expectedOutput: 'f8p'
     }],
     buggyOutputTests: [{
-      buggyFunction: 'forgetLastLetter',
+      buggyFunction: 'AuxiliaryCode.forgetLastLetter',
       messages: [
-        "It looks like your output (%s) doesn't match our expected output (%s).",
+        "It looks like your output doesn't match our expected output.",
```

Comment:
I wonder if we could say something like "In your head, try running your code on the following string: ... Does it give the correct answer?"

Otherwise this seems to hang in the air a bit -- it's not clear what the student should do. If you want to be a bit less specific at this point, you could say something like "It looks like your code contains an error. Can you find it?"

In both cases, though, it's clear to the user what they need to do next. With the current message, I was wondering if I needed to be making guesses about the test cases (i.e. reading the "interviewer's" mind).